x-pack/test/functional/apps/transform/editing.ts

```diff
@@ -152,10 +144,31 @@ export default function ({ getService }: FtrProviderContext) {
             'Frequency',
             testData.transformFrequency
           );
+
+          await transform.testExecution.logTestStep('should update the transform retention policy');
```

Comment:
The test step says it'll update the retention policy, but the code just checks the initial values. I think it would be good to actually set the retention policy to something after asserting initial values, so we know they can be saved without error.

Reply:
👍 Updated in 61ed8e4.